build.sbt

```diff
@@ -441,8 +453,7 @@ lazy val catsJVM = project
              alleycatsCore.jvm,
              alleycatsLaws.jvm,
              alleycatsTests.jvm,
-             jvm,
-             docs
```

Comment:
If remove docs from here, is it still going to be run in CI?

Reply:
Yes, it's a separate task in the Travis CI config.